settings/src/script.js

```diff
@@ -156,22 +156,15 @@ function Main( { userId } ) {
 	);
 
 	if ( 'account-status' === screen ) {
-		screenContent = (
-			<div className={ 'wporg-2fa__account-status' }>
-				<AccountStatus />
-			</div>
-		);
+		screenContent = <AccountStatus />;
 	} else if (
 		twoFactorRequiredScreens.includes( screen ) &&
-		userRecord.record[ '2fa_available_providers' ] &&
-		userRecord.record[ '2fa_revalidation' ] &&
-		userRecord.record[ '2fa_revalidation' ].expires_at <= new Date().getTime() / 1000
+		record[ '2fa_available_providers' ].includes( 'Two_Factor_Totp' ) &&
```

Comment:
Can we achieve the same by checking the size and the array and that it doesn't just contain backup codes?

Reply:
Updated